packages/react-components/react-charts-preview/library/src/components/CommonComponents/Popover.tsx

```diff
@@ -193,20 +200,9 @@ const PopoverComponent: React.FunctionComponent<IPopoverComponentProps> = React.
                   ? classes.calloutBlockContainertoDrawShapetrue
                   : classes.calloutBlockContainertoDrawShapefalse,
               )}
-              style={{
-                ...(!toDrawShape
-                  ? {
-                      borderLeft: `4px solid ${xValue.color}`,
-                      marginTop: props.XValue ? '13px' : 'unset',
-                    }
-                  : {}),
-              }}
             >
               <div className={classes.calloutlegendText}> {xValue.legend}</div>
-              <div
-                className={classes.calloutContentY}
-                style={{ color: props.color ? props.color : tokens.colorNeutralForeground1 }}
```

Comment:
where are you setting the colors for the y callout content

Reply:
colors need not be set for calloutContentY, as per v8